crates/workspace/src/workspace.rs

```diff
@@ -6474,3 +6482,264 @@ mod tests {
         });
     }
 }
+
+pub fn client_side_decorations(element: impl IntoElement, cx: &mut WindowContext) -> Stateful<Div> {
+    const BORDER_SIZE: Pixels = px(1.0);
+    let decorations = cx.window_decorations();
+
+    struct GlobalResizeEdge(ResizeEdge);
+    impl Global for GlobalResizeEdge {}
+
+    div()
+        .id("window-backdrop")
+        .bg(transparent_black())
+        .map(|div| match decorations {
+            Decorations::Server => div,
+            Decorations::Client { tiling, .. } => div
+                .bg(gpui::transparent_black())
```

Comment:
```suggestion
```
Minor detail, but the same background is already set right above :slightly_smiling_face: